mitodl/open-discussions
# Add loader to podcast page

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #2781 

#### What's this PR do?
Adds a loader for the podcasts API, reusing the placeholder images we use for a loader on the search page

#### How should this be manually tested?
Go to `/podcasts/` and verify that you see the loader briefly before the results come up.

#### Screenshots
![Screenshot from 2020-04-17 14-27-25](https://user-images.githubusercontent.com/863262/79878600-9af39880-83bb-11ea-8130-398991f63516.png)

![Screenshot from 2020-04-21 11-23-00](https://user-images.githubusercontent.com/863262/79884203-db0a4980-83c2-11ea-9cfa-cdcb102154ef.png)
